ee/clickhouse/sql/person.py

```diff
@@ -95,29 +102,40 @@
 PERSONS_DISTINCT_ID_TABLE_BASE_SQL = """
 CREATE TABLE {table_name} ON CLUSTER {cluster}
 (
-    id Int64,
     distinct_id VARCHAR,
     person_id UUID,
     team_id Int64,
-    is_deleted Int8 DEFAULT 0
+    _sign Int8 DEFAULT 1,
+    is_deleted Int8 ALIAS if(_sign==-1, 1, 0)
```

Comment:
Note: This is here for queries to work in a backwards-compatible way. We can remove this once queries are updated.